modules/memberships/class-jetpack-memberships.php

```diff
@@ -357,6 +417,17 @@ public static function get_connected_account_id() {
 		return get_option( self::$connected_account_id_option_name );
 	}
 
+	/**
+	 * Determines whether the current user can edit.
+	 *
+	 * @return bool Whether the user can edit.
+	 */
+	public static function user_can_edit() {
+		$user = wp_get_current_user();
+		// phpcs:ignore ImportDetection.Imports.RequireImports.Symbol
+		return 0 !== $user->ID && current_user_can( 'edit_post', get_the_ID() );
```

Comment:
If we want to check for admins, would checking for `manage_options` be enough here? I worry that with this approach, we may target authors as well?

Reply:
I believe we do want to target authors, since the intention is for a user editing a post to be able to preview their change on the frontend even if it is not fully configured. My use of 'admin' is confusing; I'll make sure to change that.